### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 14th of June 2022 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#770] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 14th of June 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

* Zoom link:  https://us06web.zoom.us/j/84336116610
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: [link](https://www.youtube.com/watch?v=K1iSKwHIeQ8&ab_channel=Kinvolk)
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- Community call renamed to office hours
- We're now on the Kubernetes Slack, too. This is meant to complement our presence on Matrix, not to replace it.
  https://kubernetes.slack.com/archives/C03GQ8B5XNJ
- relaxation of release cadence - from releases every ~2 weeks to ~4 weeks

## Spotlights
- Gabriel demos garm for running self-hosted github actions

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by gabriel-samfira** [comment]
Hello folks!

For the next community call, I would like to propose a demo of [garm](https://github.com/cloudbase/garm/). Garm is a GitHub actions self hosted runner manager with auto-scaling. It allows projects to use any infrastructure as a platform for GitHub self-hosted runners, allowing access to more resources or specialized hardware within GitHub workflows. This would allow Flatcar to re-purpose the current Jenkins setup and switch to GitHub actions, while leveraging the current hardware infrastructure, as well as potentially adding jobs for other CPU architectures.

Cheers,
Gabriel
